setup.cfg

```diff
@@ -7,4 +7,4 @@ all_files  = 1
 upload-dir = build/docs/html
 
 [tool:pytest]
-DJANGO_SETTINGS_MODULE = analytical.tests.settings
+DJANGO_SETTINGS_MODULE = tests.testproject.settings
```

Comment:
This references the test project relative to the project root.

It's not quite as clean as having separate separate source and test roots, but not having to deal with that complexity is probably good for now.

Reply:
This is just because `pytest` is run from the project root. In the full picture, `tox` installs `django-analytical` via `setup.py` in a virtual environment, then `pytest` is run with that environment variable. What would you do differently to further improve separation?

Reply:
For cleaner separation, I would look at a package layout `src` and `tests` as separate roots, without referring to anything from the top-level, as described here:

* https://blog.ionelmc.ro/2014/05/25/python-packaging/#the-structure
* https://hynek.me/articles/testing-packaging/

This approach has some advantages, but I think it's out of scope for this PR, and not really important in the short term ofr this project.

Reply:
I understand. The use of `src` is debated, though. Personally, I feel it creates unnecessary mental overhead. This is, I believe, inspired by the Java and C++ world where you compile code and have a more complex setup (e.g. with XML files to picture the build dependencies). It makes sense there. Luckily, with Python things are less wordy. That's what I think.